docs/topics/ksp/ksp-quickstart.md

```diff
@@ -1,12 +1,15 @@
 [//]: # (title: KSP quickstart)
 
-For a quick start, you can create your own processor or get a [sample one](https://github.com/google/ksp/tree/main/examples/playground).
+For a quick start, you can create your own processor or get a [sample one](https://github.com/google/ksp/releases/download/%kspSupportedKotlinVersion%-%kspVersion%/playground.zip).
```

Comment:
There is no sample for the latest version, so we need to keep the hardcoded version here.

Reply:
Hardcoded in [this commit](https://github.com/JetBrains/kotlin-web-site/pull/2716/commits/e93111e82f23994106cde233abc6ca08fc56cf38).